docassemble_webapp/docassemble/webapp/templates/pages/update_package.html

```diff
@@ -16,12 +18,12 @@ <h2>{{ word('Install or update a package') }}</h2>
 
 <form action="{{ url_for('update_package') }}" method="POST" class="form" role="form" enctype="multipart/form-data">
   {{ form.hidden_tag() }}
-  <h4>{{ word('You can provide a:') }}</h4>
+  <h3>{{ word('You can provide a:') }}</h3>
```

Comment:
```suggestion
  <h2 class="h3">{{ word('You can provide a:') }}</h2>
```

Reply:
Ended up changing these back to h3, but keeping the h4 class, since they are all sub headings of "Install or update a package", and shouldn't be at the same level.